docs/user-guide/cli-using-daemon-mode.md

```diff
@@ -28,16 +28,29 @@ The following steps describe how to enable daemon mode and how to configure Zowe
    zowe daemon enable
    ```
 
-   The command copies the Zowe executables for your operating system into the $ZOWE_CLI_HOME/bin (.zowe/bin) directory. The next command that you issue starts the daemon.
+   The command copies the Zowe executables for your operating system into the `$ZOWE_CLI_HOME/bin` (`.zowe/bin`) directory. The next command that you issue starts the daemon.
 2. Add the path to the Zowe executables to the environment variables for your computer. For example:
```

Comment:
Suggested rewording:
Add the path to the Zowe executable to your PATH environment variable. For example: